File: src/01-easy/the-descent/solution.cpp
#include <iostream>
using namespace std;
int main() {
    while (true) {
        int max = -1, idx = 0;
        for (int i = 0; i < 8; i++) {
            int h; cin >> h;
            if (h > max) { max = h; idx = i; }
        }
        cout << idx << endl;
    }
}

File: src/01-easy/the-descent/solution.fs
open System
[<EntryPoint>]
let main _ =
    while true do
        let mutable max = -1
        let mutable idx = 0
        for i in 0..7 do
            let h = Console.ReadLine() |> int
            if h > max then max <- h; idx <- i
        printfn "%d" idx
    0
